comp/otelcol/otlp/components/exporter/serializerexporter/serializer.go

```diff
@@ -87,6 +89,28 @@ func setupSerializer(config pkgconfigmodel.Config) {
 	config.Set("enable_payloads.service_checks", true, pkgconfigmodel.SourceDefault)
 	config.Set("enable_payloads.sketches", true, pkgconfigmodel.SourceDefault)
 	config.Set("enable_payloads.json_to_v1_intake", true, pkgconfigmodel.SourceDefault)
+
+	// Proxy Setup
+	proxyConfig := httpproxy.FromEnvironment()
+	if proxyConfig.HTTPProxy != "" {
+		config.Set("proxy.http", proxyConfig.HTTPProxy, pkgconfigmodel.SourceDefault)
+	}
+	if proxyConfig.HTTPSProxy != "" {
+		config.Set("proxy.https", proxyConfig.HTTPSProxy, pkgconfigmodel.SourceDefault)
+	}
+
+	// ProxyURL takes precedence over proxy environment variables if set
+	if cfg.HTTPConfig.ProxyURL != "" {
```

Comment:
Does the HTTPConfig separate between http/ https proxy ? Or does it not so we apply it to both ?

Reply:
Correct, HTTPConfig does not separate between http/s so we apply it for both